src/app-layer-htp.c

```diff
@@ -2898,54 +2898,6 @@ static void HTPConfigParseParameters(HTPCfgRec *cfg_prec, ConfNode *s,
                     cfg_prec->http_body_inline = 0;
                 }
             }
-        } else if (strcasecmp("swf-decompression", p->name) == 0) {
-            ConfNode *pval;
-
-            TAILQ_FOREACH(pval, &p->head, next) {
-                if (strcasecmp("enabled", pval->name) == 0) {
-                    if (ConfValIsTrue(pval->val)) {
-                        cfg_prec->swf_decompression_enabled = 1;
-                        SCLogWarning("Flash decompression is deprecated and will be removed in "
```

Comment:
Should we keep this warning @jasonish ?

Reply:
yes, but reworded that it is now removed

Reply:
Ok